src/homepageExperience/components/steps/python/InstallDependenciesSql.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {PureComponent} from 'react'
+import CodeSnippet from 'src/shared/components/CodeSnippet'
+import {SafeBlankLink} from 'src/utils/SafeBlankLink'
+
+import {event} from 'src/cloud/utils/reporting'
+
+export class InstallDependenciesSql extends PureComponent {
+  private logCopyCodeSnippet = () => {
+    event('firstMile.pythonWizard.InstallDependencies.code.copied')
+  }
```

Comment:
Do we have any new directions on events? There are some tickets floating around re: standardizing these, but not sure if we're using the eventing dictionary anymore, or whether standardizing those has been deprioritized.

Reply:
I'm not sure about this tbh, the eventing system was being changed at a time when a lot of other things ended up taking priority.